tests/FSharp.Test.Utilities/CompilerAssert.fs

```diff
@@ -259,8 +259,11 @@ and Compilation =
 
 module rec CompilerAssertHelpers =
 
-    let useTransparentCompiler = FSharp.Compiler.CompilerConfig.FSharpExperimentalFeaturesEnabledAutomatically
-    let checker = FSharpChecker.Create(suggestNamesForErrors=true, useTransparentCompiler=useTransparentCompiler)
+    let UseTransparentCompiler =
```

Comment:
Not a fan of this - we now basically check two similar things and I am getting confused on where we want to check what.  Would be good to have one switch or at least hide all but one of them.

Reply:
Also, UseTransparentCompiler vs UsesTransparentCompiler 👀

Reply:
This is the single switch which is used by tests when instantiating `FSharpChecker`. 

If you mean that it can be activated by both `FSHARP_EXPERIMENTAL_FEATURES` and it's own env variable, I thought it's more user friendly for contributors - they don't have to worry about another flag. But I had to add a specific one because `FSHARP_EXPERIMENTAL_FEATURES` seems to cause some unrelated tests to fail. 

`UseTransparentCompiler` = you're telling it what to do
`UsesTransparentCompiler` = you're asking what it's doing

Reply:
Oh I see. `PleaseUseTransparentCompiler` would be clearer but that's fine.